packages/appstore/src/components/wallet-button/wallet-button.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Icon, Text } from '@deriv/components';
+import { getWalletHeaderButtons } from 'Constants/utils';
+import { CSSTransition } from 'react-transition-group';
+import classNames from 'classnames';
+import './wallet-button.scss';
+
+type TProps = {
+    readonly button: ReturnType<typeof getWalletHeaderButtons>[number];
```

Comment:
why is readonly?

Reply:
@henry-deriv Why not? :) It's because we shouldn't modify button's settings inside of components, we should do this in utils file